go-controller/pkg/util/util.go

```diff
@@ -139,3 +141,15 @@ func IsAnnotationNotSetError(err error) bool {
 	_, ok := err.(annotationNotSetError)
 	return ok
 }
+
+// HashforOVN hashes the provided input to make it a valid addressSet or portGroup name.
+func HashForOVN(s string) string {
+	h := fnv.New64a()
+	_, err := h.Write([]byte(s))
+	if err != nil {
+		klog.Errorf("Failed to hash %s", s)
```

Comment:
should we return an empty string if there is an error?

Reply:
done